### DISCUSSION TITLE
Tume and date

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#114] [Q&A] [answered]
How do I correct time and date?

**Comment by T-Crypt** [comment]
This is done when you install Arch linux or whatever OS you are using. You need to set the UTC on the machine itself. Not through his .config files as they don't change the time format set on the device

**Reply by sudobrat** [reply]
is there a way to correct it now?

**Reply by T-Crypt** [reply, marked as answer]
timedatectl set-timezone Zone/SubZone 

for me Zone/Subzone is America/Denver

**Comment by sudobrat** [comment]
thanks a lot it worked
